src/_pytest/python.py

```diff
@@ -911,6 +912,172 @@ def hasnew(obj: object) -> bool:
     return False
 
 
+@attr.s(auto_attribs=True, eq=False)
```

Comment:
TODO: Apply suggestions from https://github.com/pytest-dev/pytest/pull/9350#discussion_r764843625.

Reply:
Right, some of the suggestions there are still relevant.